simpl/exceptions.py

```diff
@@ -121,3 +121,28 @@ class SimplConfigUnknownOption(SimplConfigException):
     For example, a specified ini file has an option with no corresponding
     config.Option.
     """
+
+
+class SimplHTTPError(SimplException):
```

Comment:
Can we combine this with https://github.com/checkmate/simpl/blob/master/simpl/rest.py#L33? Having two will cause confusion as to which to use.